### ISSUE TITLE
Setting default output value

### ISSUE THREAD

**Issue by Toma400** [TheIllusiveC4/Polymorph#153]
**Please describe the new feature or change.**

This feature would introduce that you could choose which one of the recipes will show as default (in best implementation: being recognised by crafting table or smelter, even if used without GUI [injecting items by hopper]).
This way, we don't need to constantly switch between outputs, especially if we always need/prefer to craft the second/third/etc one.

**Please describe why you want this feature.**

It is - most importantly - really comfortable to have one favourite block being pushed before the random default one. So changing default blocks would make some craftings easier, and if it was also interfering with the crafting block itself without the GUI, it would also make automation way more coherent/supported.
Also, it would make [this bug](https://github.com/TheIllusiveC4/Polymorph/issues/152) a bit less annoying.

**Comment by Toma400** [comment]
Sorry for so many bugs/features reported today, I just didn't report them at a time I first encountered them ^^'

**Comment by Pultex** [comment]
Oh yes this would let me use this mod again
as of right now I get an error saying it can not deserialize a recipe from polymorph as polymorph does not exist
this makes the world crash with a ticking world event
log is full of these can not deserialize in fabric crash log
removing polymorph fixes the issue

**Comment by TheIllusiveC4** [comment]
@Pultex That seems like an unrelated issue to this thread. Please open a new issue as a bug report and post your crash log so that I can investigate your problem further.

**Comment by AlgarDev** [comment]
Adding to this thread because this change #189 makes some mods really annoying such as GlassEssentials Renewed, which shares the recipe for the vanilla glass panels. It would be useful if you could click the recipe while pressing shift and it would make that recipe the default.

**Comment by RaeTheGit** [comment]
Yes, I second this. It would be really helpful in many situations to be able to set a default. When using e.g. Crafting Tweaks' in-inventory compression while having Ultimate Compression mod installed, too, you'll almost always get compressed nuggets, ingots, etc instead of packing them into ingots or blocks, which almost always is not what you want.
